src/modules/user/user.resolver.ts

```diff
@@ -47,39 +48,32 @@ export class UserResolver {
   }
 
   @Mutation(() => User)
-  async resetPassword(@Arg("input") input: ResetPasswordInput): Promise<User> {
-    const { email, newPassword } = input;
+  async resetPassword(
+    @Arg("resetPasswordInput") input: ResetPasswordInput
+  ): Promise<User> {
+    const updatedUser = await this.authService.resetPassword(input);
 
-    // Check if user exists
-    const user = await prisma.user.findUnique({ where: { email } });
-    if (!user) {
-      throw new Error("User not found");
-    }
-
-    // Generate new password hash
-    const salt = await bcrypt.genSalt(10);
-    const passwordHash = await bcrypt.hash(newPassword, salt);
+    return updatedUser;
+  }
 
-    // Update user's password hash
-    const updatedUser = await prisma.user.update({
-      where: { email },
-      data: { passwordHash },
-    });
+  @Mutation(() => String)
```

Comment:
You can remove the `() => String` from here as well, and also the Boolean one below.

Reply:
By removing it, I'm getting an error "NoExplicitTypeError: Unable to infer GraphQL type from TypeScript reflection system. You need to provide explicit type for 'deactivateAccount' of 'UserResolver' class."

Reply:
Oh, okay, well then nevermind! 😂